### ISSUE TITLE
About the personX_spgan

### ISSUE THREAD

**Issue by stoneyang** [Simon4Yan/VisDA2020#2]
Hi, @Simon4Yan 

Thanks for providing this to pave the way to entering this competition! 

I walked around and tried to run `python learn/train.py` and foud that the provided dataset, ie., `challenge_data.zip` has no subdirectory named `personX_spgan`. Am I missing something on this? Could you shed me some light on this? 

Thanks in advance!

**Comment by Simon4Yan** [comment]
Thanks for your attention.

The `personX_spgan` denotes the translated images from SPGAN. SPGAN conducts source-target image translation on `personX`, such that the translated images follow the distribution of the target. Thus, the Re-ID model trained on the translated images achieves high accuracy on the test set. 

The baseline should use `personX` as the training dataset. I have updated this. 

Thanks.

**Comment by fandulu** [comment]
Can we use personX_spgan in the challenge?
